paper.tex

```diff
@@ -57,7 +61,7 @@
 Please direct correspondence to the GitHub repository \\
 \url{https://github.com/evaluating-adversarial-robustness/adv-eval-paper} \\
 \noindent \\
-Last Update: 18 February, 2019. \\ %TODO CHANGE OTHER DATE TOO
+Last Update: 18 February, 2019 (revision \texttt{\version}).
```

Comment:
shouldn't the date also be updated automatically?

Reply:
That's a good point that we should discuss. I wasn't sure if this should be <date paper was compiled>, or <date last substantial change was made to the paper>. Perhaps doing this automatically would be better. What do others think?